pkg/services/sqlstore/migrations/libraryelements.go

```diff
@@ -50,4 +50,8 @@ func addLibraryElementsMigrations(mg *migrator.Migrator) {
 
 	mg.AddMigration("create "+models.LibraryElementConnectionTableName+" table v1", migrator.NewAddTableMigration(libraryElementConnectionV1))
 	mg.AddMigration("add index "+models.LibraryElementConnectionTableName+" element_id-kind-connection_id", migrator.NewAddIndexMigration(libraryElementConnectionV1, libraryElementConnectionV1.Indices[0]))
+
+	mg.AddMigration("add unique index library_element org_id_uid", migrator.NewAddIndexMigration(libraryElementsV1, &migrator.Index{
```

Comment:
Any risk of this failing? Guess all the `uid` values stored there have been auto-generated by Grafana and this migration should "just work" :smile:

Reply:
Yes, all previous `uid` where auto-generated, and you couldn't provide your own so in theory this should "just work"?.

Reply:
Yeah exactly 😄 good